README.md

```diff
@@ -23,6 +25,9 @@ This Action commits the contents of your Git tag to the WordPress.org plugin rep
 * `VERSION` - defaults to the tag name; do not recommend setting this except for testing purposes.
 * `ASSETS_DIR` - defaults to `.wordpress-org`, customizable for other locations of WordPress.org plugin repository-specific assets that belong in the top-level `assets` directory (the one on the same level as `trunk`).
 * `BUILD_DIR` - defaults to `false`. Set this flag to the directory where you build your plugins files into, then the action will copy and deploy files from that directory. Both absolute and relative paths are supported. The relative path if provided will be concatenated with the repository root directory. All files and folders in the build directory will be deployed, `.disignore` or `.gitattributes` will be ignored.
+* `SKIP_WP_VERSION` - skips the retrieval of the current WP version from wordpress.org and does not update the readme.txt's "Tested up to" line.
```

Comment:
I think my preference here would be the opposite here, that the default would NOT auto-bump the version without specific action by a maintainer.  As such, I'd say changing this to `BUMP_WP_TESTED_UP_TO` or something and that only when that is set to `true` (or otherwise present I guess) would it trigger the functionality in `deploy.sh` below.

Reply:
Also, if someone forgot to properly bump versions then they could use https://github.com/10up/action-wordpress-plugin-asset-update to correct their mistake, right?

Reply:
Sounds good, I've pushed an update to invert the logic.